docs/src/apireference.md

```diff
@@ -169,6 +169,18 @@ The value of the attribute is of type `TerminationStatusCode`.
 TerminationStatusCode
 ```
 
+### Conflict Status
+
+The `ConflictStatus` attribute indicates why the conflict finder stopped executing.
+The value of the attribute is of type `ConflictStatusCode`.
+
+```@docs
+compute_conflict!
+ConflictStatus
+ConflictStatusCode
+ConstraintConflictStatus
```

Comment:
Travis is failing because you are missing `ConflictParticipationStatusCode` here

Reply:
Fixed, thanks!